src/backend/electron/multiDownloader.ts

```diff
@@ -0,0 +1,94 @@
+import fs from "node:fs";
+import path from "node:path";
+import { createLogger } from "@/helpers/log";
+import { ProgressCallback } from "@/helpers/progressHelper";
+
+const log = createLogger("downloadFiles");
```

Comment:
意図はこう？
```suggestion
const log = createLogger("multiDownloader");
```

（このコードが書かれてるファイル名を取得できる関数とかあれば便利そう）